packages/carbon-web-components/src/components/structured-list/structured-list.ts

```diff
@@ -30,6 +30,18 @@ class BXStructuredList extends FocusMixin(LitElement) {
   @property({ attribute: 'selection-name' })
   selectionName = '';
 
+  /**
+   * Specify if structured list is condensed, default is false
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'condensed' })
+  condensed = false;
+
+  /**
+   * Specify if structured list is flush, default is false
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'flush' })
```

Comment:
```suggestion
  @property({ type: Boolean, reflect: true })
```
I think `attribute` may be redundant here?

Reply:
@emyarod I think you forgot to remove the redundant attributes

Reply:
looks like I renamed them instead of removing them 😅